### ISSUE TITLE
Filters for picking teams out with

### ISSUE THREAD

**Issue by kidoblivious** [madcowfred/padherder-issues#288]
While making out a team it would be very convenient if I could filter the list by ATT and type, in much the same way as I now can in the game.

In my imagination you have the grid of your monster pics on the right, with filter buttons at the top for the two options with the pics the game has for ATTs and Types. It would also be nice to able to sort them (asc/desc) by HP/ATK/RCV initially.
Having a toggle for sub colours would be keeping all the familiar interface options.

I suggest this as I find myself using two tabs open to pick teams out with.

**Comment by jarch3r** [comment]
I agree picking out team members from a visual grid with filters/sort would be much more user friendly and convenient than the text selection we have now and it would vastly improve my/peoples experience with the team building feature.

I'm not positive, but unfortunately I believe this has been brought up before and madcowfred was not interested in implementing it at that time.

**Comment by madcowfred** [comment]
I don't remember if this was suggested previously - I like the idea but it's also a lot of work :\

**Comment by kidoblivious** [comment]
For a less work idea, how about just filter by ATT/Type, forget grids, sorting, maybe just grey out/move to the bottom things on the lists we already have?
